docs/tpm_renders.md

```diff
@@ -0,0 +1,32 @@
+## The TPM concept
+According to the many varied pinouts necessary was design the concept of TPM 
+module which can fit the motherboard pinout. There was the consideration of 
+two kinds of the interface (SPI and LPC). The whole setup would be made of two 
+PCBs. One is the main PCB with the MCU and SPI or LPC configured socket and the 
+second is the adapter that fits the motherboards pinouts.
+![Flash](images/renders/koncepcja_TPM_adapter.svg)
+
+## The adapter
```

Comment:
I don't think this chapter is still relevant. Below in the '## Concept of slim adapter' part of the document, you present newer version of adapter `### The adapter:
<center>![Flash](images/renders/adapter.PNG)</center>`

Reply:
ok